cucumber/authenticators_oidc/features/support/authn_oidc_helper.rb

```diff
@@ -27,8 +28,16 @@ def set_id_token_user_property_variable
     set_oidc_variable("id-token-user-property", oidc_id_token_user_property)
   end
 
-  def set_oidc_variable(variable_name, value)
-    path = "cucumber:variable:conjur/authn-oidc/keycloak"
+  def set_provider_uri_variable_without_service_id(value = oidc_provider_uri)
+    set_oidc_variable("provider-uri", value, "")
+  end
+
+  def set_id_token_user_property_variable_without_service_id
+    set_oidc_variable("id-token-user-property", oidc_id_token_user_property, "")
+  end
```

Comment:
I don't think this (or the other methods with long names that wrap `set_oidc_variable`) add much value, and they do force one extra step of indirection when reading the code.  I would remove them and call `set_oidc_variable` directly from the step definitions.